docs/planteamiento.md

```diff
@@ -0,0 +1,22 @@
+# Planteamiento del proyecto
+
+## Historias de Usuario
+
+- [[HU01] Hortensia quiere encontrar a la persona adecuada para el puesto ofertado.](https://github.com/aacosa34/Keikaku/issues/7)
+
+  > Hortensia es muy indecisa con la elección de la persona adecuada para un puesto de trabajo. Tiene muchos currículums y no sabe por cuál decidirse y quiere un punto de vista más objetivo.
+
+- [[HU02] Hortensia quiere ahorrar tiempo revisando los CVs.](https://github.com/aacosa34/Keikaku/issues/3)
+
+  > Hortensia pasa muchísimo tiempo revisando uno por uno cada uno de los CVs que le llegan a una oferta de empleo revisando que cumplan los criterios exigidos, y se le hace una tarea demasiado repetitiva y agotadora.
+
+- [[HU03] Hortensia quiere ahorrarse el tener que buscar el teléfono o email de contacto dentro de aquellos currículums que sean aceptados para agilizar el proceso de concertar las entrevistas.](https://github.com/aacosa34/Keikaku/issues/6)
+  > Hortensia pierde demasiado tiempo buscando dentro de cada currículum los datos de contacto de cada posible candidato, tarea que también se le hace bastante repetitiva. Además, cada persona tiene un diseño y distribución de los datos distinto en su currículum, por lo que tiene que hacer un esfuerzo buscando dónde se encuentran estos datos.
+
+## Milestones
+
+- [[M1] Modelo para almacenar los datos personales contenidos en los CVs.](https://github.com/aacosa34/Keikaku/milestone/7)
```

Comment:
Un modelo tampoco es un producto. Tienes que decir cómo se empaqueta para que lo sea efectivamente.

Reply:
Imagino que en éste milestone tengo que especificar en la descripción qué datos almaceno y de qué manera, ¿verdad?

Reply:
No, eso lo debe hacer quien lo programe. Tienes que  definir el envoltorio, cómo se va a entregar ese modelo. ¿En una servilleta? ¿En un diagrama UML? ¿O de qué forma?
Pista: lo hemos explicado en clase.
También: si esta línea no aparece como "outdated", es que no lo has cambiado. Debéis hacer cambios en las líneas comentadas para superar el objetivo.